documentation/modules/proc-kafka-connect-mounting-volumes.adoc

```diff
@@ -25,13 +33,17 @@ metadata:
   name: mysecret
 type: Opaque
 stringData:
-  connector.properties: |-
-    dbUsername: my-user
+  connector.properties: |- <1>
+    dbUsername: my-user <2>
     dbPassword: my-password
 ----
+<1> The connector configuration in properties file format.
+<2> Database _username_ and _password_ properties used in the configuration.
 
 . Create or edit the Kafka Connect resource.
-Configure the `FileConfigProvider` in the `config` section and the `externalConfiguration` section of the `KafkaConnect` or `KafkaConnectS2I` custom resource to reference the secret.
++
+Configure the configuration provider in the `config` section and the `externalConfiguration` section of the `KafkaConnect` or `KafkaConnectS2I` custom resource to reference the secret.
```

Comment:
This sentence is quite tdifficult to parse! You could change it as follows:

In the `config` section and the `externalConfiguration` section of the `KafkaConnect` or `KafkaConnectS2I` custom resource, configure the configuration provider to reference the secret.